status/reporters_test.go

```diff
@@ -110,7 +110,7 @@ func (c *MockK8sClient) List(ctx context.Context, list client.ObjectList, opts .
 	return c.err
 }
 
-func (c *MockK8sClient) Get(ctx context.Context, key types.NamespacedName, obj client.Object) error {
```

Comment:
@dirgim @amisstea I update this file because I meet error as below when I run `make test`
```
make test
/Users/hongliu/HAS/integration-service-revert-kcp/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/Users/hongliu/HAS/integration-service-revert-kcp/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
# github.com/redhat-appstudio/integration-service/status_test
vet: status/reporters_test.go:198:56: cannot use mockK8sClient (variable of type *MockK8sClient) as client.Reader value in argument to status.NewGitHubReporter: *MockK8sClient does not implement client.Reader (wrong type for method Get)
		have Get(ctx context.Context, key "k8s.io/apimachinery/pkg/types".NamespacedName, obj client.Object) error
		want Get(ctx context.Context, key "k8s.io/apimachinery/pkg/types".NamespacedName, obj client.Object, opts ...client.GetOption) error
```
After I update function `get`, `make test` can pass. Could you check if this update is right? Thank you!

Reply:
This looks fine. This is a byproduct of the upgrade of `sigs.k8s.io/controller-runtime` from `v0.12.1` to `v0.13.1`. See: https://github.com/kubernetes-sigs/controller-runtime/commit/6533c61cea9b083b68d8379fbe88c05b269cc4a2

Reply:
Thanks Alex. Yes, it worked on the old packages.